src/components/charts/list/DiscoverCharts.tsx

```diff
@@ -364,43 +363,48 @@ function DiscoverChartList() {
+                                                                                state.charts.length === 0
+                                                                            }
+                                                                            addChart={addChart}
+                                                                            showDescription={!isGrid}
+                                                                            subtractChart={subtractChart}
+                                                                            onClick={(chartId) =>
+                                                                                state.charts.length === 0
+                                                                                    ? history.push(
+                                                                                          `${url}/chart/${chart.id}`,
+                                                                                      )
+                                                                                    : selectChart(chartId)
+                                                                            }
+                                                                        />
+                                                                    ))}
+                                                            </div>
+                                                        </>
                                                     ) : (
-                                                        <EmptyChartGroup
-                                                            title={'No matching charts'}
-                                                            removeLearnMore={true}
-                                                            image={emptyImage}
-                                                            onClickViewChartButton={clearSearch}
-                                                            subTitle={`We couldn't find any matching results`}
-                                                            styles={{ height: '300px', justifyContent: 'center' }}
-                                                        />
+                                                        <ChartEmptyState onClickViewChartButton={clearSearch}>
+                                                            <InfoColourBar
+                                                                message={'Can’t find what you’re looking for?'}
+                                                                classname={'br-4 bw-1 bcv-1 ev-2 dc__mxw-300 bcv-1 fs-12 pl-12 pr-12'}
+                                                                Icon={Help}
+                                                                iconClass={'fcv-5 h-20'}
+                                                                linkText={'Try refetching connected chart repos or connect a chart repository'}
```

Comment:
Directly pass it as string. No need to wrap around {}

Reply:
done